src/utils/styles.ts

```diff
@@ -23,6 +23,7 @@ export const theme = {
     text: '#607d8b',
     gray: '#777',
     black: '#000000',
+    grey: { '1': '#eee', '2': '#8a8a8a' },
```

Comment:
would prefer `gray1` and `gray2` instead of an object

Reply:
ok